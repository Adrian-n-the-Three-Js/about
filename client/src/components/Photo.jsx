import React from 'react';
import styled from 'styled-components';

console.log('photo');

const Image = styled.img`
  display: block;
  max-width: 100%;
  object-fit: cover;
`;

const Photo = (props) => {
  console.log('photo url', props.url)
  return (
    <Image
      className="photo"
      src={props.url}
      onClick={() => { props.toggleModal(props.album) }}
    />
  );
};

export default Photo;
