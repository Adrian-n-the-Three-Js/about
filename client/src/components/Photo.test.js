import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Photo from './Photo.jsx';

describe ('Photo', () => {
  xit('should render correctly', () => {
    const Photoomponent = shallow(<Photo />);
    expect(PhotoComponent).toMatchSnapshot();
  });
});
