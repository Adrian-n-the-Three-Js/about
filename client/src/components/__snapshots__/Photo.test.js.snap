// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo should render correctly 1`] = `
<Photo__Image
  className="photo"
/>
`;
